taskRequests/details/style.css

```diff
@@ -261,3 +324,53 @@ body {
     max-width: 40%;
   }
 }
+
+.requestor_details_modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-black-shade-70percent);
+  right: 0;
+}
+
+.requestor_details_modal_content {
+  background-color: var(--color-light-gray);
+  margin: 10% auto;
+  padding: 2rem;
+  border: 1px solid var(--color-gray-variant2);
+  width: 25%;
+  text-align: center;
+  right: 10%;
```

Comment:
I don't see a position property in this class, so is this right even doing something?

Reply:
yeah the position property is on its parent. this class is used for the content inside the modal i.e the proposed start date , end date and description. I think its inheriting the position property from its parent